tests/register.spec.coffee

```diff
@@ -1,9 +1,6 @@
-requestMock = require 'requestmock'
-mockery = require 'mockery'
-
-requestMock.log(false)
-mockery.enable(warnOnUnregistered: false)
-mockery.registerMock('request', requestMock)
+Promise = require 'bluebird'
+global.Promise = Promise
```

Comment:
Why are you setting the global promise here?  It will mean the test environment is different from the standard environment used